spinoso-string/src/impls.rs

```diff
@@ -172,35 +184,35 @@ impl From<&str> for String {
 impl From<String> for Vec<u8> {
     #[inline]
     fn from(s: String) -> Self {
-        s.buf
+        s.inner.as_vec().clone()
```

Comment:
This conversion consumes the original string, so we don't need to clone here.

```suggestion
        s.into_vec()
```